src/main/java/seedu/loyaltylift/logic/commands/AddCustomerCommand.java

```diff
@@ -18,22 +19,24 @@ public class AddCustomerCommand extends Command {
 
     public static final String COMMAND_WORD = "addc";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a customer to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a customer to the address book. \n"
             + "Parameters: "
+            + "[" + PREFIX_CUSTOMER_TYPE + "ind|ent] "
```

Comment:
Not sure what is ideal here but since all other attributes are capitalised like NAME, PHONE etc, should IND and ENT be capitalised as well when shown in the invalid command format?

Reply:
I left it as lowercase to indicate it is a set of items that the user has to choose from, this is to separate from other attributes where it's just a placeholder. 
Anyways, I've found the proper formatting style and have updated both the docs and usage message here. Hopefully it resolves this issue!